src/components/Search/SearchResultsListItem.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import Checkbox from '@mui/material/Checkbox';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import PropTypes from 'prop-types';
+
```

Comment:
Does it make sense to move `truncateHash` and `getLastRevision` in the utils folder?